apple/LiveMarkdownModule.mm

```diff
@@ -1,40 +1,57 @@
+#import "LiveMarkdownModule.h"
+
+#import <RNLiveMarkdown/RuntimeDecorator.h>
+
 #ifdef RCT_NEW_ARCH_ENABLED
+#import <RNLiveMarkdown/MarkdownCommitHook.h>
+#endif // RCT_NEW_ARCH_ENABLED
 
+#import <React/RCTBridge+Private.h>
+
+#ifdef RCT_NEW_ARCH_ENABLED
 #import <React/RCTScheduler.h>
 #import <React/RCTSurfacePresenter.h>
+#endif // RCT_NEW_ARCH_ENABLED
 
-#import "MarkdownCommitHook.h"
-#import "MarkdownShadowFamilyRegistry.h"
-#import "RCTLiveMarkdownModule.h"
+#import <jsi/jsi.h>
 
+using namespace facebook;
 using namespace expensify::livemarkdown;
 
 // A turbomodule used to register the commit hook
 // I think this is the easiest way to access the UIManager, which we need to
 // actually register the hook
 
-@implementation RCTLiveMarkdownModule {
+@implementation LiveMarkdownModule {
   BOOL installed_;
+#ifdef RCT_NEW_ARCH_ENABLED
   std::shared_ptr<MarkdownCommitHook> commitHook_;
   __weak RCTSurfacePresenter *surfacePresenter_;
+#endif // RCT_NEW_ARCH_ENABLED
 }
 
-RCT_EXPORT_MODULE(@"LiveMarkdownModule")
+RCT_EXPORT_MODULE()
 
-- (NSNumber *)install {
-  if (!installed_ && surfacePresenter_ != nil) {
-    RCTScheduler *scheduler = [surfacePresenter_ scheduler];
+RCT_EXPORT_BLOCKING_SYNCHRONOUS_METHOD(install)
+{
+  RCTCxxBridge *cxxBridge = (RCTCxxBridge *)self.bridge;
+  jsi::Runtime &rt = *(jsi::Runtime *)cxxBridge.runtime;
+  expensify::livemarkdown::injectJSIBindings(rt);
 
-    commitHook_ = std::make_shared<MarkdownCommitHook>(scheduler.uiManager);
-    installed_ = YES;
-  }
-  return @1;
+#ifdef RCT_NEW_ARCH_ENABLED
+  RCTScheduler *scheduler = [surfacePresenter_ scheduler];
+  commitHook_ = std::make_shared<MarkdownCommitHook>(scheduler.uiManager);
+  installed_ = YES;
```

Comment:
I believe this can be removed (and the property as well)

Reply:
Removed in 62bc84e